tripal_chado/src/api/tripal_chado.db.api.php

```diff
@@ -661,15 +658,15 @@ function chado_autocomplete_dbxref($db_id, $string = '') {
   }
   $sql = "
     SELECT dbxref_id, accession
-    FROM {dbxref}
+    FROM {1:dbxref}
     WHERE db_id = :db_id and lower(accession) like lower(:accession)
     ORDER by accession
     LIMIT 25 OFFSET 0
   ";
-  $results = chado_query($sql, [
+  $results = \Drupal::service('tripal_chado.database')->query($sql, [
     ':db_id' => $db_id,
     ':accession' => $string . '%',
-  ]);
+  ])->execute();
```

Comment:
```suggestion
  ]);
```
In this case we don't want execute.